source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1266,6 +1267,28 @@ message TransformationPermutePhiOperands {
 
 }
 
+message TransformationPropagateInstructionDown {
+
+  // Propagates an instruction from |block_id| into its successors.
+  // Concretely, the transformation clones the propagated instruction
+  // into some of the predecessors of |block_id| and removes the original
+  // instruction. Additionally, OpPhi instruction may be added to make sure
+  // that the transformation can be applied in various scenarios.
+
+  // Id of the block to propagate an instruction from.
+  uint32 block_id = 1;
+
+  // A fresh id for an OpPhi instruction.
+  uint32 phi_fresh_id = 2;
+
+  // A map from the id of some successor of the |block_id| to the fresh id.
+  // The map contains a fresh id for at least every successor of the |block_id|.
+  // Every fresh id in the map corresponds to the result id of the clone,
+  // propagated into the corresponding successor block.
+  repeated UInt32Pair successor_id_to_fresh_id = 3;
```

Comment:
If overflow ids are available, they can be used to provide fresh ids for successors that are not present in the map.

Could you remark on that here, make sure the transformation uses overflow ids if it runs out of ids, and include at least one test that requires them?  (See the test I did for TransformationOutlineFunctions as an example.)

Reply:
Fixed in 69b41e9.